### PR TITLE
JOB: Product and community lead

### PR DESCRIPTION
**Pull request by choldgraf**
This is a draft post + job description for our new role, the Product and Community Lead. We'll plan to post this sometime next week, but I'm opening up a draft for now in case others would like to take a look and provide feedback.

### Preview

- [Our jobs page is here](https://deploy-preview-93--cocky-booth-e7ed17.netlify.app/jobs/)
- [This job posting is here](https://deploy-preview-93--cocky-booth-e7ed17.netlify.app/jobs/open/product-community-lead-2022/)
- [The blog post is here](https://deploy-preview-93--cocky-booth-e7ed17.netlify.app/blog/2022/job-product-community-lead/)

### To do

- [x] Finish blog post
- [x] Include the job posting and update the `/jobs` page
- [x] Finish the CS&S application portal
- [x] Agree on a date to post
- [x] Post the job!

closes https://github.com/2i2c-org/meta/issues/261

### CODE REVIEW

**Comment by damianavila** [comment]
I did a quick iteration and I really like the way it is written, @choldgraf.
It actually looks like a tempting position!! Great job!

**Comment by choldgraf** [comment]
### Update: most content is ready

I've got most of the content ready for a review. I think there's still more to update, but I would love feedback from folks if they have any thoughts in the meantime.

Our goal should be to post this by the end of the week, unless something major happens in the news that means we should bump it to next week.

**Comment by celsobessa** [comment]
Just a quick note, although the jobs page summary states it is a fully remote position, the job posting itself doesn't, and it might be useful.

**Comment by choldgraf** [comment]
Good point @celsobessa - I have clarified this in the new info box at the top of the job posting, and also provided a link to the main "jobs" page to make that content more discoverable.

**Comment by colliand** [comment]
The job posting has a `_know _` that should be `know`.

The blog post has a sentence: "However, the reality is that this role will ultimately get to shape the nature of this work within 2i2c themselves. " that I think should read: "However, the reality is that the person in this role will ultimately get to shape the nature of their work within 2i2c." 

Apart from those two small edits, this looks excellent!

File: content/jobs/open/product-community-lead-2022.md
Comment (colliand) [inline content/jobs/open/product-community-lead-2022.md]: `_kind _` should be `kind` or `_kind_` if italics was intended.

File: content/blog/2022/job-product-community-lead/index.md
Comment (colliand) [inline content/blog/2022/job-product-community-lead/index.md]: "However, the reality is that the person in this role will ultimately get to shape the nature of their work within 2i2c.

File: content/blog/2022/job-product-community-lead/index.md
Comment (arfon) [inline content/blog/2022/job-product-community-lead/index.md]: Perhaps say something about prioritisation and making tradeoffs?

File: content/jobs/open/product-community-lead-2022.md
Comment (arfon) [inline content/jobs/open/product-community-lead-2022.md]:
```suggestion
**Foster deeper external relationships and engagements** with key communities of practice and partners that are important for 2i2c’s strategy, such as the [Pangeo Project](https://pangeo.io) (which is a key stakeholder for this role).
```

I _think_ this needs to be a proper URL?

File: content/jobs/open/product-community-lead-2022.md
Comment (arfon) [inline content/jobs/open/product-community-lead-2022.md]: > * Experience with distributed and complex open source communities and ability to communicate and collaborate effectively with these communities.About us page

The _About us page_ bit of this looks like a typo?

File: content/jobs/_index.md
Comment (sgibson91) [inline content/jobs/_index.md]:
```suggestion
Instead, we ask team members to communicate their intentions early with one another, and discuss with the team if anybody has concerns that a person's absence will harm our ability to meet our goals.
```

typo

File: content/jobs/open/product-community-lead-2022.md
Comment (sgibson91) [inline content/jobs/open/product-community-lead-2022.md:63]:
```suggestion
This is the first position of its kind within 2i2c, and it will have freedom to shape its strategy and role with the organization. It should oversee the evolution and growth of this _kind_ of work within 2i2c, and develop organizational strategy for how to maximize our impact. Below are a few ways that this might occur (both independently and in partnership with others at 2i2c):
```

**Review by sgibson91 — APPROVED** [review]
LGTM modulo the above comments are addressed!

File: content/jobs/open/product-community-lead-2022.md
Comment (choldgraf) [inline content/jobs/open/product-community-lead-2022.md]:
```suggestion
This is the first position of its kind within 2i2c, and it will have freedom to shape its strategy and role with the organization. It should oversee the evolution and growth of this _kind_ of work within 2i2c, and develop organizational strategy for how to maximize our impact. Below are a few ways that this might occur (both independently and in partnership with others at 2i2c):
```

File: content/jobs/open/product-community-lead-2022.md
Comment (choldgraf) [inline content/jobs/open/product-community-lead-2022.md]:
```suggestion
* Experience with distributed and complex open source communities and ability to communicate and collaborate effectively with these communities.
```

File: content/blog/2022/job-product-community-lead/index.md
Comment (choldgraf) [inline content/blog/2022/job-product-community-lead/index.md]:
```suggestion
However, the reality is that the person in this role will ultimately get to shape the nature of their work within 2i2c. As a young organization, there is a lot of flexibility for creativity and experimentation in bringing new skillsets into our organization. We hope that the person who fills this role will be excited about growing a culture of team-based approaches to our engineering and collaborations, and to share this culture with the communities that we serve.
```

File: content/blog/2022/job-product-community-lead/index.md
Comment (choldgraf) [inline content/blog/2022/job-product-community-lead/index.md]:
```suggestion
This role is kind-of like a **product manager** - their job is to understand the communities that we work with, and to use this to help us make decisions about what to build, how to present it, and how to engage with stakeholders. They will need to help us prioritize the work that is most impactful for our mission, and help us navigate trade-offs in the evolution of our services.
```

**Review by damianavila — APPROVED** [review]
I know there might be little typos or wording to modify yet, but this one has, in general, the green light for me, so approving it.

**Comment by choldgraf** [comment]
OK, I believe I've addressed all of the comments people have given, and made some minor clean-ups to the language. I think that we are ready to go here.

If I do not hear any objections, then **I plan to post this job on Tuesday around 8:00AM US/Pacific**. 🚀 

Either way, I still welcome feedback, comments, and suggestions!

**Pull request merged**
